tests/functional/test_buildah.py

```diff
@@ -145,7 +148,8 @@ def test_build_failure(tmpdir):
     target_image_name = "registry.example.com/ab-test-" + random_word(12)
     target_image_tag = "oldest"
     target_image = f"{target_image_name}:{target_image_tag}"
-    target_failed_image = target_image + "-failed"
+    timestamp = datetime.datetime.now().strftime(TIMESTAMP_FORMAT)
+    target_failed_image = timestamp + "-" + target_image + "-failed"
```

Comment:
after the build command, you can use `inspect` command (with no args, which will show the last build) and get the image name from the output - I'd then make sure the name matches the expect format, likely using a regex